recipes/libfec/meta.yaml

```diff
@@ -1,42 +1,47 @@
 {% set name = "libfec" %}
-{% set version = "2.1.1" %}
+{% set version = "2.1.1+20161003" %}
+{% set deb_version = "1.0+20161003" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: 'https://github.com/quiet/libfec/archive/master.zip'
```

Comment:
Since the contents of this zip can change when the master branch is updated, I switched to pulling the source from Debian since there is no other tarball to get it from.

Reply:
Yes, better! In direct github, the checksum is constraining the version, but ends in a broken package when the master.zip is updated so much better using this way!